CrossPlatformDesktopProject/CrossPlatformDesktopProject/CollisionHandler/CollisionResponse.cs

```diff
@@ -66,22 +66,35 @@ private void BuildDictionary()
                 .SelectMany(s => s.GetTypes())
                 .Where(p => typeOfItems.IsAssignableFrom(p));
 
-            //Enemy on player
+            
             foreach (CollisionSides side in Enum.GetValues(typeof(CollisionSides)))
             {
+                
                 foreach (Type enemySubject in enemyTypes)
                 {
+                    //Enemy on player
                     commandMap.Add(new Tuple<Type, Type, CollisionSides>(enemySubject, playerType, side), typeof(TakeDamageCommand));
+                    //Player on enemy
                     commandMap.Add(new Tuple<Type, Type, CollisionSides>(typeof(Sword), enemySubject, side), typeof(EnemyTakeDamageCommand));
+                    commandMap.Add(new Tuple<Type, Type, CollisionSides>(typeof(Bow), enemySubject, side), typeof(EnemyTakeDamageCommand));
+                    commandMap.Add(new Tuple<Type, Type, CollisionSides>(typeof(Boomerang), enemySubject, side), typeof(EnemyTakeDamageCommand));
+                    commandMap.Add(new Tuple<Type, Type, CollisionSides>(typeof(Smoke), enemySubject, side), typeof(EnemyTakeDamageCommand));
                 }
+
+                //Obstacle on player
                 foreach (Type obstacleSubject in obstacleTypes)
                 {
                     commandMap.Add(new Tuple<Type, Type, CollisionSides>(obstacleSubject, playerType, side), typeof(ResetCommand));
                 }
+                commandMap.Add(new Tuple<Type, Type, CollisionSides>(typeof(Bomb), playerType, side), typeof(ResetCommand));
```

Comment:
Why is the bomb an automatic death/game reset? And should this be `typeof(Smoke)`?

Reply:
Oh nevermind that's not what reset means